src/templating/data.ts

```diff
@@ -1,7 +1,8 @@
 import got from 'got';
 import { getDebugFunction } from '../utils/logger';
 import { stripIndent } from 'common-tags';
-
+import { readLocalEnvFile } from '../config/utils/env';
+import { OutgoingHttpHeaders } from 'http';
```

Comment:
got() options required headers of type `OutgoingHttpHeaders`, but I wonder if this was the correct way of handling it or I'm just not using TypeScript correctly?

Reply:
Yeah this should be fine!